### ISSUE TITLE
A11y_Azure Tools for VS Code [Benchmark]_Default Screen_Keyboard: Unable to perform "Create New Project" and "Create New Function" using keyboard instead getting error message

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2690]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yMAS; #A11yTCS; #Benchmark-Azure Tools for VS Code-Win-Feb2021; #AILimited; #DesktopApp; #WCAG2.1.1; #Keyboard; #Win10;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till "Create New Project" and "Create Function" and press enter on them
8. Verify user is able to perform expected operation or not.

### Actual Result:
Unable to perform "Create New Project" and "Create New Function" using keyboard instead getting error message

### Expected Result:
User should be able to perform "Create New Project" and "Create New Function" using keyboard instead of getting error message

### User Impact:
Keyboard users will not perform all operations if they are not accessible using keyboard

### Attachment for Reference:
